code/obj/machinery/manufacturer.dm

```diff
@@ -643,21 +636,9 @@
 				src.queue -= src.queue[operation]
 				begin_work(1)//pesky exploits
```

Comment:
```suggestion
				begin_work() //pesky exploits
```
arg defaults to 1 anyways. alternately `begin_work(TRUE)` if you'd prefer to be clear.